contrib/syntax-checking/packages.el

```diff
@@ -147,5 +148,12 @@
     :commands helm-flyspell-correct
     :init (evil-leader/set-key "Sc" 'helm-flyspell-correct)))
 
+(defun syntax-checking/post-init-evil ()
+  (add-to-list 'evil-motion-state-modes 'flycheck-error-list-mode)
+  (evil-define-key 'motion flycheck-error-list-mode-map
+    (kbd "RET") #'flycheck-error-list-goto-error
```

Comment:
just curious, why not just `"RET" #'flycheck-error-list-goto-error`? Why the (kbd ) wrapper around "RET"?

I also added an `evil-leader` key to hide the error list, namely `eL` and I find it very useful to close the flycheck error list

(defun spacemacs/flycheck-hide-list-errors ()
  "Hide the error list for the current buffer."
  (interactive)
  (let ((buffer (get-buffer flycheck-error-list-buffer)))
    (when buffer
      (let ((window (get-buffer-window buffer)))
        (when window
          (unless (flycheck-overlays-at (point))
            (quit-window nil window)))))))